tests/e2e/analytical_mode/workloads.yaml

```diff
@@ -0,0 +1,14 @@
+analytical_mode_cluster: &analytical_mode_cluster
+  cluster:
+    main:
+      args: ["--bolt-port", "7687", "--log-level=TRACE"]
+      log_file: "analytical_mode.log"
+      setup_queries: []
```

Comment:
If I was missing something in the previous comment, this could also be a place where we could create an initial graph in the database, but I think the fixture is a better place to do that in this case.